server/routes/request.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { createRequest, getRequests, updateRequest } = require("../controllers/request");
+
+router.route("/").post(protect, createRequest);
+router.route("/").get(protect, getRequests);
+router.route("/").put(protect, updateRequest);
```

Comment:
```suggestion
router.route("/:requestId").put(protect, updateRequest);
```
We'll need a path param to identify the specific resource we wish to identify/modify